bodhi/tests/server/functional/test_generic.py

```diff
@@ -12,8 +12,13 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
+import copy
+
 from pyramid.testing import DummyRequest
 
+from webtest import TestApp
```

Comment:
Since webtest is a third party library it does make sense to be grouped with ```pyramid```, so let's remove the space between them.